### PR TITLE
Res2net model support

### PR DESCRIPTION
**Pull request by stevenhillis**
Include support for Res2Net models, as used in microsoft 2020 diarization system: https://arxiv.org/pdf/2010.11458.pdf

### CODE REVIEW

**Comment by Shane-pe** [comment]
Cheers, thank you for your contribution.

File: models/Res2NetSE34.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        x = self.conv1(x)
+        x = self.relu(x)
+        x = self.bn1(x)
+
+        x = self.layer1(x)
+        x = self.layer2(x)
+        x = self.layer3(x)
+        x = self.layer4(x)
+
+        x = x.reshape(x.sixe()[0], -1, x.size()[-1])
```
Comment (009deep) [inline models/Res2NetSE34.py:97]: You mean size, right!!

**Comment by 009deep** [comment]
Good attempt.  Couple of suggestion:
1. Model you used  is Res2Net50 and not 34, so rename would be good.
2. It'd be good to parameterize  use of SE  layer, basewidth and  scale values.
3. Run code to make sure, no bugs.
Also, to note that Microsoft used Res2Net blocks  but architecture was different form one presented in Res2Net paper.

File: models/Res2NetSE50.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        self.conv1 = nn.Conv2d(1, num_filters[0], kernel_size=3, stride=1, padding=1)
+        self.relu = nn.ReLU(inplace=True)
+        self.bn1 = nn.BatchNorm2d(num_filters[0])
+
+        self.layer1 = self._make_layer(block, num_filters[0], layers[0])
+        self.layer2 = self._make_layer(block, num_filters[1], layers[1], stride=(2, 2))
+        self.layer3 = self._make_layer(block, num_filters[2], layers[2], stride=(2, 2))
+        self.layer4 = self._make_layer(block, num_filters[3], layers[3], stride=(2, 2))
+
+        self.instancenorm = nn.InstanceNorm1d(n_mels)
```
Comment (dvisockas) [inline models/Res2NetSE50.py:34]: I msft res2net [paper](https://arxiv.org/pdf/2007.02480.pdf) for speaker verification they use mean-only nomalization.

**Pull request closed without merging**